### DISCUSSION TITLE
podman v. 2.2.1 play kube: readiness?

### DISCUSSION THREAD

**Discussion by sophonet** [containers/podman#9119] [Q&A]
Hi guys,

on my server, I have created a yaml file with podman generate kube, describing a pod with two services. One (mariadb) needs to be started first, and the second (wordpress) needs to wait until the first is up and running in order to launch properly. What is the appropriate way of how to achieve this with podman play kube? My impression is that initContainers (the Kubernetes approach) is not (yet) supported?

Thanks and best regards,

sophonet

**Comment by vrothberg** [comment]
Thanks for reaching out!  To be honest, I am not very familiar with the Kube YAML but if there's something missing, we'll take a look.

Can you elaborate which feature or YAML field you're looking for?

**Comment by mheon** [comment]
There is an enhancement request open for initContainers - https://github.com/containers/podman/issues/6480

**Comment by sophonet** [comment]
I see, so I guess currently it is not possible to start pods in such a way that multiple containers are launched in the correct order and when dependent ones are ready for being used. So I guess currently this needs to be manually done with e.g. a shell script and in-between commands for waiting...
